hudi-utilities/src/test/java/org/apache/hudi/utilities/testutils/sources/AbstractDFSSourceTestBase.java

```diff
@@ -79,14 +85,24 @@ public void setup() throws Exception {
   @AfterEach
   public void teardown() throws Exception {
     super.teardown();
+    dfs.delete(new Path(dfsRoot), true);
+    dfs.delete(new Path(dfsArchivePath), true);
   }
 
   /**
    * Prepares the specific {@link Source} to test, by passing in necessary configurations.
    *
    * @return A {@link Source} using DFS as the file system.
+   * @param defaults
    */
-  protected abstract Source prepareDFSSource();
+  protected abstract Source prepareDFSSource(TypedProperties defaults);
+
+  /**
+   * Prepares the specific {@link Source} to test.
+   */
+  protected Source prepareDFSSource() {
+    return prepareDFSSource(new TypedProperties());
```

Comment:
if we follow the proposed changes in tests, we don't need to make any changes to these tests.